roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
+    set_fact:
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+      openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+
+  - name: Normalize openshift_release
+    set_fact:
+      openshift_release: "{{ openshift_release | string | regex_replace('^v', '') }}"
+    when: openshift_release is defined
+
+- name: Ensure a valid deployment type has been given |
```

Comment:
Ansible jams on all of the parameters in the output after the name. So it's the difference in readability between:

`TASK [openshift_inventory : Ensure a valid deployment type has been given msg=Please set openshift_deployment_type to one of:`

and:

`TASK [openshift_inventory : Ensure a valid deployment type has been given | msg=Please set openshift_deployment_type to one of:`

In the second case it's visually clearer where you can stop reading. If there were some way (aside from no_log which also suppresses useful debug output) to not show the action parameters in the header I would want to do that. No one else seems too concerned about the output ansible gives the user but... to me, it would be nice if it were as clear as possible.